test/eoftest/eoftest_runner.cpp

```diff
@@ -41,8 +41,20 @@ void from_json(const json::json& j, EOFValidationTest::Case& o)
         throw std::invalid_argument{"code is invalid hex string"};
     o.code = *op_code;
 
-    if (const auto it_initcode = j.find("initcode"); it_initcode != j.end())
+    if (const auto it_kind = j.find("containerKind"); it_kind != j.end())
+    {
+        const auto v = it_kind->get<std::string>();
+        if (v == "INITCODE")
+            o.kind = ContainerKind::initcode;
+        else if (v == "RUNTIME")
+            o.kind = ContainerKind::runtime;
```

Comment:
RUNTIME is default and never explicitly listed it seems. Up to a unittest to cover this, I'd add a separate backburner issue for this and the unknown case too.